lansa-vmss-windows-autoscale-sql-database/createUiDefinition.json

```diff
@@ -209,8 +209,7 @@
                         },
                         "constraints": {
                             "required": true
-                        },
-                        "visible": "[equals(steps('SecurityConfig').keyVaultEnabled, 'false')]"
```

Comment:
Why have you removed this line?

Reply:
Since the default value of cert pswd is removed now, we need to provide some value for this param in the test.parameter.json in both senarios, that is, if it is used or if keyvault is used.
Thats why I have made cert pswd param visible in both scenarios ( whether keyvault is used or externally passed cert string is used)

Reply:
I see. Then the situation is we must specify a default value for the password because we do not want to show a parameter to the user. Its not like we actually are specifying a default password. Only syntactically. The reviewer will have to accept it.

So back out the UI change and put the password default value back in.

Reply:
Sure.